contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -619,8 +723,11 @@ contract SetConfig is SetUp {
   }
 }
 
+// allow NOPs to withdraw balances made before disableOffchainNOPsOffchain is called
```

Comment:
this comment feels a little out of place. can we put it next to the test that it is describing?